src/pip/_internal/wheel.py

```diff
@@ -1174,7 +1170,6 @@ def build(
                     python_tag=python_tag,
                 )
                 if wheel_file:
-                    build_success.append(req)
                     self.wheel_filenames.append(
```

Comment:
This should also be moved with `build_success.append()` since its purpose is to communicate the set of files that were generated (and which should be available in the output directory for processing after `pip wheel` finishes).

Reply:
done